src/com/sun/ts/tests/signaturetest/SigTestDriver.java

```diff
@@ -58,7 +58,30 @@
   private static final String DEBUG_FLAG = "-Debug";
 
   private static final String FORMATPLAIN_FLAG = "-FormatPlain";
-
+  
+  private static final String EXCLUDE_JDK_CLASS_FLAG = "-IgnoreJDKClass";
+
+  private static String[] excludeJdkClasses = {
+          "java.util.Map",
+          "java.lang.Object",
+          "java.io.ByteArrayInputStream",
+          "java.io.InputStream",
+          "java.lang.Deprecated",
+          "java.io.Writer",
+          "java.io.OutputStream",
+          "java.util.List",
+          "java.util.Collection",
+          "java.lang.instrument.IllegalClassFormatException",
+          "javax.transaction.xa.XAException",
+          "java.lang.annotation.Repeatable",
+          "java.lang.InterruptedException",
+          "java.lang.CloneNotSupportedException",
+          "java.lang.Throwable",
+          "jakarta.resource.spi.SecurityException",
```

Comment:
A little late.. Is this supposed to be `java.lang.SecurityException`? If not, why is a Jakarta EE class in the exclude list?

Reply:
Thanks @hussainnm for raising this!  I'm checking into it now.